docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -684,4 +684,138 @@ The default is 9000.
   -V, --version
 ``` 
 
-Displays the version and exits. 
+Displays the version and exits.
+
+### validators-external-signer-public-keys
+
+```bash tab="Syntax"
+--validators-external-signer-public-keys=<KEY>[,<KEY>...]
+```
+
+```bash tab="Command Line"
+
+--validators-external-signer-public-keys=0xa99a...e44c,0xb89b...4a0b
+```
+
+```bash tab="Environment Variable"
+TEKU_VALIDATORS-EXTERNAL-SIGNER-PUBLIC-KEYS=0xa99a...e44c,0xb89b...4a0b
+```
+
+```bash tab="Configuration File"
+validators-external-signer-public-keys=[
+  "0xa99a...e44c",
+  "0xb89b...4a0b"]
+```
+
+List of validator public keys used by an external signer (for example Eth2Signer).
+
+### validators-external-signer-timeout
+
+```bash tab="Syntax"
+--validators-external-signer-timeout=<INTEGER>
+```
+
+```bash tab="Command Line"
+--validators-external-signer-timeout=2000
+```
+
+```bash tab="Environment Variable"
+TEKU_VALIDATORS-EXTERNAL-SIGNER-TIMEOUT=2000
+```
+
+```bash tab="Configuration File"
+validators-external-signer-timeout=2000
+```
+
+Timeout in milliseconds for requests to the external signer. Default is 1000.
+
+### validators-external-signer-url
+
+```bash tab="Syntax"
+--validators-external-signer-url=<URL>
+```
+
+```bash tab="Command Line"
+--validators-external-signer-url=http://localhost:9000
+```
+
+```bash tab="Environment Variable"
+TEKU_VALIDATORS-EXTERNAL-SIGNER-URL=http://localhost:9000
+```
+
+```bash tab="Configuration File"
+validators-external-signer-url="http://localhost:9000"
+```
+
+URL on which the external signer (for example Eth2Signer) is running.
+
+### validators-key-file
+
+```bash tab="Syntax"
+--validators-key-file=<PATH_TO_FILE>
+```
+
+```bash tab="Command Line"
+--validators-key-file=/home/me/me_node/key.yaml
+```
+
+```bash tab="Environment Variable"
+TEKU_VALIDATORS_KEY_FILE=/home/me/me_node/key.yaml
+```
+
+```bash tab="Configuration File"
+validators-key-file="/home/me/me_node/key.yaml"
+```
+
+YAML formatted file to load unencrypted validator keys from.
+
+The file can be created when generating validators using the `teku validator generate --keys-output-path` option.
+
+### validators-key-files
+
+```bash tab="Syntax"
+--validators-key-files=<FILENAME>[,<FILENAME>...]...
+```
+
+```bash tab="Command Line"
+--validators-key-files=validator_0xa245...58cf.json,validator_0xb880...1f09.json
+```
+
+```bash tab="Environment Variable"
+TEKU_VALIDATORS_KEY_FILES=validator_0xa245...58cf.json,validator_0xb880...1f09.json
+```
+
+```bash tab="Configuration File"
+validators-key-files=[
+  "validator_0xa245...58cf.json",
+  "validator_0xb880...1f09.json"]
+```
+
+List of encrypted BLS12-381 keystore files to load the validator keys from.
+
+The files can be created when generating validators using the `teku validator generate --keys-output-path` option.
```

Comment:
Same comment as above

Reply:
This option by default will generate encrypted BLS keystore files/password files as the "encrypted" flag defaults to true.